view-creator-framework/src/main/java/org/hypertrace/core/viewcreator/pinot/PinotViewCreatorConfig.java

```diff
@@ -57,4 +57,11 @@ public class PinotViewCreatorConfig {
   // REST API
   /////////////
   public static final String PINOT_REST_URI_TABLES = "tables";
+
+  // text index config keys
+  public static final String TEXT_INDEX_CONFIG_COLUMN = "column";
+  public static final String TEXT_INDEX_CONFIG_SKIP_PRIOR_SEGMENTS = "skipPriorSegments";
```

Comment:
Seeing both `skipExistingSegments` and `skipPriorSegments`?

Reply:
Done. Align with Align with : https://docs.pinot.apache.org/configuration-reference/table#field-config-list.